### PR TITLE
Update suggested action given by update-cluster to stop compute fleet

### PR DESCRIPTION
**Pull request by tilne**
Signed-off-by: Tim Lane <tilne@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cli/src/pcluster/config/update_policy.py [hunk 1]
```diff
@@ -116,7 +116,8 @@ def __eq__(self, other):
         "modify-ebs-volume",
     ),
     "pcluster_stop": lambda change, patch: (
-        f"Stop the cluster with the following command:\npcluster stop {patch.cluster_name}"
+        "Stop the cluster with the following command:\npcluster update-compute-fleet --cluster-name  "
```
Comment (enrico-usai) [inline cli/src/pcluster/config/update_policy.py:119]: I'd say stop the compute fleet. 

Then is STOP_REQUESTED valid for both Slurm and Batch? If it's not the case we can just say stop the compute fleet, without mentioning the command.

Comment (tilne) [reply cli/src/pcluster/config/update_policy.py:119]: > is STOP_REQUESTED valid for both Slurm and Batch?

I assume so, based on the fact that `UpdatePolicy.AWSBATCH_CE_MAX_RESIZE` has uses the same reason.

Comment (enrico-usai) [reply cli/src/pcluster/config/update_policy.py]: Unfortunately it is not true: https://github.com/aws/aws-parallelcluster/blob/develop/cli/src/pcluster/api/controllers/cluster_compute_fleet_controller.py#L66

Comment (tilne) [reply cli/src/pcluster/config/update_policy.py]: so does it need to be DISABLED instead of STOP_REQUESTED for batch?

Comment (tilne) [reply cli/src/pcluster/config/update_policy.py]: > without mentioning the command

i think not including  the command to run defeats the purpose of providing suggested actions

**Review by enrico-usai — APPROVED** [review]

**Review by enrico-usai — APPROVED** [review]

**Pull request merged**